pkg/server/options/embeddedetcd.go

```diff
@@ -44,6 +45,7 @@ func (e *EmbeddedEtcd) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&e.PeerPort, "embedded-etcd-peer-port", e.PeerPort, "Port for embedded etcd peer")
 	fs.StringVar(&e.ClientPort, "embedded-etcd-client-port", e.ClientPort, "Port for embedded etcd client")
 	fs.Int64Var(&e.WalSizeBytes, "embedded-etcd-wal-size-bytes", e.WalSizeBytes, "Size of embedded etcd WAL")
+	fs.BoolVar(&e.ForceNewCluster, "embedded-etcd-force-new-cluster", e.ForceNewCluster, "Starts a new cluster even if previously started")
```

Comment:
maybe `Starts a new cluster from existing data restored from a different system` ?

Reply:
How does etcd describe that flag?

Reply:
I have it 1:1 from etcd https://github.com/etcd-io/etcd/blob/main/server/embed/config.go#L342 :-)

Reply:
For inspiration from https://chromium.googlesource.com/external/github.com/coreos/etcd/+/v2.0.13/Documentation/admin_guide.md:

> To restore a backup using the procedure created above, start etcd with the -force-new-cluster option and pointing to the backup directory. This will initialize a new, single-member cluster with the default advertised peer URLs, but preserve the entire contents of the etcd data store. Continuing